telco-ran/configuration/README.md

```diff
@@ -0,0 +1,14 @@
+# Reference configuration
+
+## Structure
+This directory contains three components of the reference configuration
+ - The `source-crs` tree contains the CRs which form the foundation of
```

Comment:
I would slightly rephrase all the below to something like:
```
- source-crs contain the CRs..
- extra-manifests-builder is used to create...
```
to avoid using `folder`/`tree` all the time.